proto/ibc/core/channel/v1/tx.proto

```diff
@@ -81,6 +81,7 @@ message MsgChannelOpenTry {
   string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
   // in the case of crossing hello's, when both chains call OpenInit, we need
   // the channel identifier of the previous channel in state INIT
+  // this field has been deprecated and will be ignored by core IBC because now we don't need to protect against crossing hellos anymore.
```

Comment:
I think if you add `Deprecated:` as a prefix it can be flagged by tools like gostaticcheck, seeing as unfortunately golang doesn't have any built in deprecation feature.

```suggestion
  // Deprecated: this field has been deprecated and will be ignored by core IBC because now we don't need to protect against crossing hellos anymore.
```